tools/please_go_filter/please_go_filter.go

```diff
@@ -0,0 +1,45 @@
+// Package main implements please_go_filter, a tool for filtering go source files.
+// This command parses source files and file names to determine whether they
+// match the build constraints that you'd usually encounter with `go build`.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"go/build"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+var taglist = flag.String("tags", "", "Comma-separated list of extra build tags to apply as build constraints.")
```

Comment:
The rest of the tools and whatnot don't use the standard `flag` package, although this one doesn't need any of the more advanced features of go-flags so I don't have very strong feelings on it.

Reply:
I started out using Please's `cli` package and, therefore, go-flags, but this was a dependency cycle because `cli` depends on please_go_filter for its own build.  It seemed simpler to only depend on built-in packages rather than work some way around this.

Reply:
Right, of course :)